components/datetime/tests/patterns/structs/time_zones.rs

```diff
@@ -4,12 +4,12 @@
 
 use icu_datetime::time_zone;
 use serde::{Deserialize, Serialize};
-use tinystr::TinyStr8;
+use tinystr::{TinyAsciiStr, TinyStr8};
 
 #[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct TimeZoneConfig {
-    pub time_zone_id: Option<String>,
-    pub metazone_id: Option<String>,
+    pub time_zone_id: Option<TinyAsciiStr<8>>,
+    pub metazone_id: Option<TinyAsciiStr<4>>,
```

Comment:
I explicitly changed these to be `TinyAsciiStr` because it was causing the tests to fail without `--features serde` enabled.

The alternative would be to make a test-only, always-serde-enabled type to read the data-driven test JSON and then convert that into the real type as we've done before. But I don't see a point because `TinyAsciiStr` essentially is that already.

Reply:
OK fair enough. I approved the PR so feel free to hit the merge button :)